tests/core/tools/manual/getmousebutton.html

```diff
@@ -0,0 +1,34 @@
+<body>
```

Comment:
We don't need a `body` tag here, can be safely removed..

Reply:
Done 👍